docs/reference/intro.asciidoc

```diff
@@ -155,81 +184,146 @@ You can use a combination of dynamic and explicit mapping on the same index.
+
+[TIP]
+====
+<<esql,{esql}>> is our new piped query language (and compute engine) that is initially mainly focused on time-series data like logs and metrics.
+====
 
 [discrete]
 [[search-data]]
-==== Searching your data
+==== Searching and filtering data
 
-The {es} REST APIs support structured queries, full text queries, and complex
-queries that combine the two. Structured queries are
-similar to the types of queries you can construct in SQL. For example, you
-could search the `gender` and `age` fields in your `employee` index and sort the
-matches by the `hire_date` field. Full-text queries find all documents that
-match the query string and return them sorted by _relevance_&mdash;how good a
-match they are for your search terms.
+{es} support a wide range of search techniques including:
 
-In addition to searching for individual terms, you can perform phrase searches,
-similarity searches, and prefix searches, and get autocomplete suggestions.
+* <<full-text-queries,*Full-text search*>>. Search text that has been analyzed and indexed to support full-text search based on relevance.
+* <<keyword,*Keyword search*>>. Search for exact matches using `keyword` fields.
+* <<semantic-search-semantic-text,*Semantic search*>>. Search `semantic_text` fields using dense or sparse vector search on embeddings generated in your {es} cluster.
+* <<knn-search,*Vector search*>>. Search for similar dense vectors using the kNN algorithm for embeddings generated outside of {es}.
+* <<geo-queries,*Geospatial search*>>. Search for locations and calculate spatial relationships using geospatial queries.
 
-Have geospatial or other numerical data that you want to search? {es} indexes
-non-textual data in optimized data structures that support
-high-performance geo and numerical queries.
+Learn about the full range of queries supported by the <<query-dsl,Query DSL>>.
 
-You can access all of these search capabilities using {es}'s
-comprehensive JSON-style query language (<<query-dsl, Query DSL>>). You can also
-construct <<sql-overview, SQL-style queries>> to search and aggregate data
-natively inside {es}, and JDBC and ODBC drivers enable a broad range of
-third-party applications to interact with {es} via SQL.
+You can also filter data using the Query DSL.
+Filters enable you to include or exclude documents by retrieving documents that match specific field-level criteria.
+A query that uses the `filter` parameter indicates <<filter-context,filter context>>.
+
+{esql} also has powerful filtering capabilities.
 
 [discrete]
 [[analyze-data]]
 ==== Analyzing your data
 
-{es} aggregations enable you to build complex summaries of your data and gain
-insight into key metrics, patterns, and trends. Instead of just finding the
-proverbial “needle in a haystack”, aggregations enable you to answer questions
-like:
+{es} enables a host of use cases based on aggregations and analytics.
+These include real-world applications like:
```

Comment:
links? :) pls

Reply:
@shainaraskas I'm unsure about this list — feels repetitious versus the main use cases overview. thoughts?